os/drivers/usbhost/uvc/usbhost_uvc_video.c

```diff
@@ -0,0 +1,1190 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+* Included Files
+****************************************************************************/
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <semaphore.h>
+#include <assert.h>
+#include <errno.h>
+#include <debug.h>
+
+#include <tinyara/kmalloc.h>
+#include <tinyara/usb/usb.h>
+#include <tinyara/usb/usbhost.h>
+#include "usbhost_uvc.h"
+
+static void usbhost_transfer_callback(FAR void *arg, ssize_t nbytes);
+FAR struct usbhost_uvc_state_s *g_usbuvc_handoff;
```

Comment:
If `g_usbuvc_handoff` is used only in this file, please add static.

Reply:
Done